app/helpers/session_timeout_warning_helper.rb

```diff
@@ -11,6 +11,13 @@ def warning
     (Figaro.env.session_timeout_warning_seconds || 30).to_i
   end
 
+  def timeout_refresh_url
```

Comment:
this method seems sufficiently complicated that it might be worth having a unit test?